docker-files/Dockerfile.ubi.min

```diff
@@ -1,7 +1,7 @@
 # Dockerfile to build PowerStore CSI Driver
 # based on UBI-minimal image
 # Requires: RHEL host with subscription
-FROM registry.access.redhat.com/ubi8/ubi-minimal@sha256:3aa3f379a81013bd3264faa0af87d201cdaa5981050d78c567b48fdfd5b38bb8
+FROM registry.access.redhat.com/ubi8/ubi-minimal@sha256:21504085e8d740e62b52573fe9a1a0d58a3e7dba589cac69734ad2fa81d66635
```

Comment:
Please put a comment describing the image for which this sha key corresponds to. For ex for this:
# ubi8/ubi-minimal:8.5-230.1645809059

Reply:
done